pkgs/applications/networking/instant-messengers/deltachat-desktop/default.nix

```diff
@@ -21,12 +21,12 @@
 
 let
   libdeltachat' = libdeltachat.overrideAttrs (old: rec {
-    version = "1.112.7";
+    version = "1.112.8";
     src = fetchFromGitHub {
       owner = "deltachat";
       repo = "deltachat-core-rust";
       rev = version;
```

Comment:
Did this actually build at the time this was merged?

It looks like upstream revs here are `v${version}` https://github.com/deltachat/deltachat-core-rust/releases/tag/v1.112.8 and this fetch 404's locally for me (noticed while nixpkgs-review'ing another PR)

Reply:
(Let me know if I'm just doing a dumb. Here was the output from the build in nixpkgs-review:)

```
trying https://github.com/deltachat/deltachat-core-rust/archive/1.112.8.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0    14    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
curl: (22) The requested URL returned error: 404
error: cannot download source from any mirror
```

Reply:
Okay, I don't think I'm just holding it wrong. Hydra is failing too for deltachat-desktop: https://hydra.nixos.org/build/217089656

Reply:
Oops, my bad. I just copied the hash from the expression for libdeltachat.